src/command_modules/azure-cli-ml/azure/cli/command_modules/ml/service/realtime.py

```diff
@@ -249,14 +249,29 @@ def realtime_service_run_local(service_name, input_data, verbose):
 
 
 # TODO - not yet used
-def realtime_service_scale(context, args):
+def realtime_service_scale(service_name, num_replicas, context=cli_context):
     """Scale a published realtime web service."""
 
     if context.in_local_mode():
         print("Error: Scaling is not supported in local mode.")
         print("To scale a cluster based service, switch to cluster mode first: az ml env cluster")
         return
 
+    elif context.env_is_k8s:
+        try:
+            num_replicas = int(num_replicas)
+        except ValueError:
+            print("The -z option must be a non-negative integer.")
```

Comment:
Doesn't actually validate the value is non-negative.
As discussed in chat, let's limit to 1-17.